### PR TITLE
feat(metadata models): update layer metadata models

### PR DESCRIPTION
**Pull request by shimoncohen**

### CODE REVIEW

**Review by cl-shaharshavit-rnd — CHANGES_REQUESTED** [review]
some models are missing from typescript types as they are not defined as "base"

File: Schema/yaml/geojson.yaml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        type: string
+        enum:
+          - GeometryCollection
+        description:
+      geometries:
+        type: array
+        items:
+          $ref: "#/definitions/Geometry"
+
+responses:
```
Comment (cl-shaharshavit-rnd) [inline Schema/yaml/geojson.yaml:148]: if we want to have models for errors they should be in separate file

File: Schema/yaml/layerMetadata.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
+openapi: "3.0.1"
+info:
+  title: Layer metadata definitions
+  version: 1.0.0
+servers:
```
Comment (cl-shaharshavit-rnd) [inline Schema/yaml/layerMetadata.yaml:5]: remove servers

**Review by cl-shaharshavit-rnd — CHANGES_REQUESTED** [review]

File: Schema/models/LayerMetadata-StatusFields.d.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { LayerMetadata } from "./LayerMetadata";
```
Comment (asafMasa) [inline Schema/models/LayerMetadata-StatusFields.d.ts:1]: shouldn't this file name be in camel case - relevant for all the repo

**Review by asafMasa — CHANGES_REQUESTED** [review]
🚀

File: Schema/models/LayerMetadata-StatusFields.d.ts (hunk 3 above)
Comment (asafMasa) [reply Schema/models/LayerMetadata-StatusFields.d.ts:1]: how " where the linter permits '

Comment (shimoncohen) [reply Schema/models/LayerMetadata-StatusFields.d.ts]: Changed

File: Schema/yaml/layerMetadata.yaml (hunk 2 above)
Comment (shimoncohen) [reply Schema/yaml/layerMetadata.yaml]: Removed

File: Schema/yaml/geojson.yaml (hunk 1 above)
Comment (shimoncohen) [reply Schema/yaml/geojson.yaml]: Removed

**Review by asafMasa — APPROVED** [review]

**Review by cl-shaharshavit-rnd — APPROVED** [review]

**Pull request merged**